test/utils.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const defaultPGOptions = {
+  PGUSER: 'postgres',
+  PGPASSWORD: '',
+  PGHOST: 'localhost',
+  PGDATABASE: 'test_globalstorage_database',
+  PGPORT: 5432,
+};
+
+for (const opt in defaultPGOptions) {
+  if (!process.env[opt]) process.env[opt] = defaultPGOptions[opt];
+}
```

Comment:
I don't like that file require will modify process.env, could you put this in a function?